tests/product/test_connectors.py

```diff
@@ -43,30 +42,33 @@ def setup_cluster_assert_connectors(self):
         self._assert_connectors_loaded([['system'], ['tpch']])
 
     @attr('smoketest')
-    def test_basic_connector_add_remove(self):
+    def test_connector_add_remove(self):
         self.setup_cluster_assert_connectors()
-
         self.run_prestoadmin('connector remove tpch')
-        self.run_prestoadmin('server restart')
         self.assert_path_removed(self.cluster.master,
                                  os.path.join(constants.CONNECTORS_DIR,
                                               'tpch.properties'))
-        self._assert_connectors_loaded([['system']])
-        for host in self.cluster.all_hosts():
-            self.assert_path_removed(host,
-                                     os.path.join(constants.REMOTE_CATALOG_DIR,
-                                                  'tcph.properties'))
 
+        # test add connectors from directory with more than one connector
         self.cluster.write_content_to_host(
             'connector.name=tpch',
             os.path.join(constants.CONNECTORS_DIR, 'tpch.properties'),
             self.cluster.master
         )
+        self.cluster.write_content_to_host(
+            'connector.name=jmx',
+            os.path.join(constants.CONNECTORS_DIR, 'jmx.properties'),
+            self.cluster.master
+        )
         self.run_prestoadmin('connector add')
```

Comment:
How does it know what connector to add? Or does it default to adding everything inside its connector directory?